### PR TITLE
SR Debugger Test Suite (Mostly)

### PR DESCRIPTION
**Pull request by okariJC**
This is my SR Debugger Test Suite with a little bit of interface integration code so I could make sure that it should work properly.

I had to move my Temp / Fake IdentityManager from the Temp Folder, into plugins because plugin classes couldn't access it

### CODE REVIEW

File: Assets/Plugins/Jamcity/JamCityID/Scripts/Controller/PopupInitializer.cs [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+﻿using gs;
+using Temp.gs.id;
+using System.Linq;
+using UnityEngine;
 using JamCityID.ScriptableData;
+using System.Collections.Generic;
 
 namespace JamCityID
 {
     public class PopupInitializer : MonoBehaviour
```
Comment (okariJC) [inline Assets/Plugins/Jamcity/JamCityID/Scripts/Controller/PopupInitializer.cs]: This class is not event close to final... So I left the `TODO:`s in here.

File: Assets/Local/Scripts/Mock/IdentityManagerOptions.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+using System.ComponentModel;
+using JetBrains.Annotations;
+using JamCityID.ScriptableData;
+
+namespace JamCityID.Debug
+{
+    [System.Serializable]
+    public class IdentityManagerOptions : INotifyPropertyChanged, ISerializationCallbackReceiver
+    {
+        private const string CATEGORY = "Logged In Info Ideninty";
```
Comment (chrissJC) [inline Assets/Local/Scripts/Mock/IdentityManagerOptions.cs]: Type: `Identity`

File: Assets/Local/Scripts/Mock/DebuggerIdentityManager.cs [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        }
+
+        public IData ActiveGames
+        {
+            get { return GetEntries(activeGames); }
+        }
+
+        public void Sync(Action<IData> success, Action<APIError> failure)
+        {
+            var option = new CallOption();
```
Comment (danielcashsgn) [inline Assets/Local/Scripts/Mock/DebuggerIdentityManager.cs]: Micro optimization:  Most of these calls have the same first line.  If you made a function to return a new option, you could turn most of them into one liners.

Comment (okariJC) [reply Assets/Local/Scripts/Mock/DebuggerIdentityManager.cs]: I might do that a little later.. good idea 😄

**Pull request merged**